### ISSUE TITLE
Starred categoryre-opens automatically. 

### ISSUE THREAD

**Issue by whitespacecode** [leggett/simplify#805]
Whenever i close the Starred category it re-opens automatically. 
Could Simplify remember the action that i did ?
![image](https://github.com/leggett/simplify/assets/37139936/c4d12192-04a7-4067-b151-abc6c29307c2)

---
Simplify v.3.1.13 configuration: aAX appNavOff avatarsInList avatarsZoom bundleInbox caA caE caI caL caS cfC cfF cfO cfR cfU composeActions composeFab dateGroup defaultTheme emailtracker_gmail fontList fontMsg fontNav gaDelete gaRemove groupHideFirst groupInbox groupOtherLists hideAds hideEmptySections hideExtWarn hideFileIcons hideInboxHideAllLink hideLabelChips hideListCount hideMsgCount hideMsgReplies hideTabIcons importantBadge inInbox inList invertCompose izBg23 lightTheme listWidthXXLg lowDensity maDone maPin matYou msgWidthSm nPane navOpen newNav newUI reminders rightImportance rightLabels rightStar sendLater simplify simplifyPrefsOpen-

**Comment by leggett** [comment]
Does it reopen when you reload Gmail? Or does it do it sooner?

**Comment by whitespacecode** [comment]
> Does it reopen when you reload Gmail? Or does it do it sooner?

I notice it when i start the day, not sure if i have a habit to refresh the page manually.
Because it definitely happens when you refresh the page.

**Comment by megaroeny** [comment]
For some reason this has been happening to me today. Clicking the "Dismiss" or "Hide" actions don't do anything.

I have to open the context menu and toggle off/on the "Hide when empty" option. Then after refreshing the tab, it displays once again.

```
Simplify v.3.1.27 u.0 configuration: aAX addOnsOpen avatarsInList avatarsZoom bundleInbox caA caI caL cfC cfF cfO cfR cfU composeActions composeFab defaultTheme extensions fontList fontMsg fontNav geminiMin genSumMin hideAds hideEmptySections hideExtWarn hideFileIcons hideImportance hideInboxHideAllLink hideLabelChips hideListCount hideMsgCount hideMsgReplies hideTabIcons importantBadge inInbox inList inboxZero invertAddons invertCompose izBg20 lightTheme listWidthLg lowDensity maDone maPin matYou msgWidthSm nPane newNav newUI oeGemini reminders rightImportance rightLabels rightStar sendLater simplify simplifyCanary simplifyPrefsOpen unreadDot - 

System: Mozilla/5.0 (Macintosh Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 - gmail.pinto-server_20241125.05_p0 - Window: 1638 x 1126 (119pct zoom) - Language: en
```

The only setting that I was messing with was the "Inbox type", switching between "Important" and "Starred" (my default). Maybe going between those triggered something to break.

**Comment by leggett** [comment]
Let me see if I can recreate the issue.

To confirm, you have a section that is empty and set to be hidden when empty. But upon refreshing, the section reappears and is expanded even though it is empty and was collapsed before?

**Comment by megaroeny** [comment]
Yup, here's a screen recording which should help:

https://github.com/user-attachments/assets/866a70a1-536e-4d0b-b551-ef0176b55b89

**Comment by leggett** [comment]
Very helpful, thanks. I think that might be a Gmail bug but it could be me -- stranger things have happened.

If you get a moment, can you disable Simplify in your browser and see if the problem persists?

**Comment by megaroeny** [comment]
> If you get a moment, can you disable Simplify in your browser and see if the problem persists?

I tried disabling it and refreshing the tab, and it appears to have mixed results. At first it didn't go away (if I disabled within the Simplify menu). After disabling + reenabling the actual extension, it appears to be working again. 🤷🏼

**Comment by leggett** [comment]
Ok, good to hear as I couldn't reproduce the issue on my end. 

I will say that Gmail has gotten a little more buggy with updating the list correctly when either sections are changed or the state of an email would move the email from one section to another. For example, I just emptied out my "Unread" section which was set to be hidden when empty. 

I then marked a message as unread and the Unread section didn't get added back. I checked the DOM and Gmail has simply not added it back at all. Only after reloading Gmail does the email reappear.

But all of this is the opposite of your original bug where the empty section kept being shown. Very strange stuff. 

P.S. And yes, disabling Simplify in the Simplify menu is temporary (it will automatically re-enable on reloading Gmail). I should add a banner that asks if you want to disable Simplify across reloads when the extension is disabled.
